paybybot3/notifs.py

```diff
@@ -24,5 +24,5 @@ def _create_apprise_object(services):
     return apobj
 
 def _send_notification(apobj, title, body, tag):
-    notify_type = TAG_MAPPING_NOTIFY_TYPE.get(tag[0]) if tag else None
+    notify_type = TAG_MAPPING_NOTIFY_TYPE.get(tag) if tag else None
```

Comment:
`notify_type = TAG_MAPPING_NOTIFY_TYPE.get(tag) if tag else None` could just be `notify_type = TAG_MAPPING_NOTIFY_TYPE.get(tag)`